### PR TITLE
Adapt Laravel docs for version 11.0

### PR DESCRIPTION
**Pull request by cleptric**
Not to be merged before March 12th.

### CODE REVIEW

File: docs/platforms/php/guides/laravel/index.mdx [hunk 1]
```diff
@@ -17,17 +17,28 @@ Install the `sentry/sentry-laravel` package:
 composer require sentry/sentry-laravel
 ```
 
-Enable capturing unhandled exception to report to Sentry by making the following change to your `App/Exceptions/Handler.php`:
+Enable capturing unhandled exception to report to Sentry by making the following change to your `bootstrap/app.php`:
```
Comment (vivianyentran) [inline docs/platforms/php/guides/laravel/index.mdx:20]: Should we call out that this is now a different file? Do users need to do anything with `App/Exceptions/Handler.php` if they previously had it?

**Review by vivianyentran — APPROVED** [review]
Looks good! Thanks for adding the docs for this version!

File: docs/platforms/php/guides/laravel/index.mdx (hunk 1 above)
Comment (cleptric) [reply docs/platforms/php/guides/laravel/index.mdx:20]: This is not a change in the SDK but instead in the framework. But the old way will still work for people upgrading from Laravel 10 to Laravel 11.

**Review by stayallive — APPROVED** [review]
Looks good to me 💪

**Pull request merged**